packages/config/src/node/loadConfig.ts

```diff
@@ -15,7 +15,7 @@ export async function loadConfig(configPath?: string): Promise<unknown> {
     configPath = await resolveConfigPath(TEMP_CONFIG);
     return (await import(configPath)).default;
   } finally {
-    rmSync(TEMP_CONFIG);
+    rmSync(TEMP_CONFIG, { force: true });
```

Comment:
why is `force: true` needed here?

Reply:
if something within `try` throws, the TEMP_CONFIG may not exist, which will cause rmSync to throw, which will swallow the original error. force makes rmSync do nothing if the file doesnt exist